staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -54,6 +54,15 @@ const (
 	// Allows us to enable anonymous auth for only certain apiserver endpoints.
 	AnonymousAuthConfigurableEndpoints featuregate.Feature = "AnonymousAuthConfigurableEndpoints"
 
+	// owner: @stlaz @tkashem @dgrisonnet
```

Comment:
it's uncommon for the owners to not be the PR authors, these guy acknowledged?

Reply:
It's ok to remove me from here

Reply:
@stlaz  since you acked, i would like to keep you as an owner, is that okay?